docs/source/contributing/roadmap.md

```diff
@@ -0,0 +1,54 @@
+# The repo2docker roadmap
+
+This roadmap collects "next steps" for the project. The goal is to
+communicate where the current priorities are and where the project is heading.
+It is not a list aimed at limiting contributions to what is listed here. If
+something is not listed here please do bring it up in a new issue or start
+work on it.
+
+Each "next step" should fit into one of the following three categories:
+
+* **now**, concrete/actionable step that is ready for someone to start work on.
+These might be items that have a link to an issue or more abstract like
+"decrease typos and dead links in the documentation"
+* **soon**, less concrete/actionable step that is going to happen soon,
+discussions around the topic are coming close to an end at which point it can
+move into the "now" category
+* **later**, abstract ideas or tasks, need a lot of discussion or
+experimentation to shape the idea so that it can be discussed. Should also
+contain concrete/actionable steps that have been postponed on purpose
+(these are steps that could be in "now" but the decision was taken to work on
+them later)
+
+The roadmap will get updated as time passes (next review by 1st December).
+This means this list should not be exhaustive, it should only represent
+the "top of the stack" of ideas. It should
+not function as a wish list, collection of feature requests or todo list.
+For those please create a
+[new issue](https://github.com/jupyter/repo2docker/issues/new).
+
+The roadmap should give the reader an idea of what is happening next, what needs
+input and discussion before it can happen and what has been postponed.
+
+
+## Guiding thought for the next months
```

Comment:
It isn't really a roadmap though, more like "thought for the day" but for the next while

Reply:
Change the title but not to roadmap. What do you think of the new one? My aim was to have a sentence that summarises the vision for the next few months. SOmething that could change over time from "strong and stable" to "more users at all costs" or "features, features, features" etc

Reply:
At some point, one needs to put a stake in the ground and create a roadmap. I guess that I'm not understanding the concept of "thought".

Perhaps make the title and add some expectations for the community (current best guess and subject to change): 

repo2docker Roadmap (WIP - subject to change)
Date: 11-xx-2018

*Then add some guidance for the reader.*

 As of Nov 2018, the "next" items are planned for the next major release. There's always the possibility that an item may not make it into the next release, but this is our best estimate as of today.

"soon" and "later" are provided to give community members a sense of future priorities as of this point in time.

Reply:
I guess we could call it a "Planning Roadmap"

Reply:
What is the downside to calling it a roadmap? Provided that we signal clearly that this is not a "promised feature release list", I don't see the problem w/ using that word. To the contrary, "roadmap" is a commonly used pattern in many open projects and I think it will be a quick way to signal to others what (could) come in the future.

Reply:
I am confused. The whole document is the roadmap, my comment was about this particular section.

Reply:
you're right, I didn't look at the context carefully enough, I'm fine w/ current wording